etc/regolith/sway/environment

```diff
@@ -0,0 +1,5 @@
+# For using firefox with wayland
```

Comment:
Kindly remove these for now

Reply:
I'd like to begin testing this session from a built package so we'll address these app specific configs later.